config/launch/standard/bot.launch

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env -S goby_launch -L -P -d 1000
 
-[kill=SIGTERM] gpsd $(../../gen/bot.py gpsd)
```

Comment:
Why is this removed? It's required for the lat/lon input from the simulator to goby_gps